src/index.html

```diff
@@ -11,6 +11,45 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+
+    <form
+      class="container large"
+      action=""
+      method="GET"
+      data-qa="big"
+    >
+      <img
+        class="icon ilarge"
```

Comment:
```suggestion
        class="icon icon--large"
```
Давай будемо використовувати модифікатори в назвах класів. `ilarge` - не дуже інтуїтивна назва